src/Rotations.jl

```diff
@@ -63,6 +63,11 @@ end
 
 injectivity_radius(M::Rotations, x) = π*sqrt(2.0)
 
+"""
+    PolarRetraction
+
+Retraction on the rotations manifold using the polar method.
+"""
```

Comment:
In the long run, the formula would be nice, at least in the md in the docs, but I'd also prefer it here.

Reply:
I've added some docs, similar to what manopt.jl has.